### PR TITLE
Some more places with remote security roles

### PR DESCRIPTION
**Pull request by pbarry25**
Three more companies with some open remote security roles. Happy to address any feedback!

Also, thank you for fixing my broken markdown link in the previous PR I submitted. I'll make sure to check my fixes locally on each PR going forward. 🙇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Auth0 | All | Full remote okay | [Link](https://auth0.com/careers) | ? |
 | bill.com |  | Some areas | [Link](https://www.bill.com/about-us/jobs) | ? |
 | Bishop Fox |  | Some areas | [Link](https://www.bishopfox.com/jobs/) | ? |
+| Bitdefender | Some | Full remote (Regional) | [Link](https://www.bitdefender.com/company/job-opportunities/) | ? |
```
Comment (pbarry25) [inline README.md:34]: They currently have at least [one remote security position in the U.S.](https://bitdefender.csod.com/ux/ats/careersite/1/home/requisition/401?c=bitdefender), I believe I've seen some remote security positions in Romania listed on their site in the past...

File: README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Auth0 | All | Full remote okay | [Link](https://auth0.com/careers) | ? |
 | bill.com |  | Some areas | [Link](https://www.bill.com/about-us/jobs) | ? |
 | Bishop Fox |  | Some areas | [Link](https://www.bishopfox.com/jobs/) | ? |
+| Bitdefender | Some | Full remote US | [Link](https://www.bitdefender.com/company/job-opportunities/) | ? |
```
Comment (pbarry25) [inline README.md:34]: Example listing: https://bitdefender.csod.com/ux/ats/careersite/1/home/requisition/401?c=bitdefender

File: README.md [hunk 3]
```diff
@@ -92,6 +93,7 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Inquest | Some | Full US remote okay | [Link](https://inquest.net/) | ? |
 | InstaCart | All | Full Remote US | [Link](https://instacart.careers/current-openings/) | ? |
 | Intel | Some | Depending on the position | [Link](https://jobs.intel.com/) | ? |
+| Jamf | Some | Full remote US | [Link](https://www.jamf.com/about/careers/) | ? |
```
Comment (pbarry25) [inline README.md:96]: Example listing: https://www.jamf.com/about/careers/jobs/?gh_jid=5151638004

File: README.md [hunk 4]
```diff
@@ -142,6 +144,7 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Samsara | Some | Full remote | [Link](https://www.samsara.com/company/careers/) | ? |
 | SecurityScoreCard | Some areas | Full remote | [Link](https://securityscorecard.com/company/careers) | ? |
 | Semmle | Some areas | Full remote | [Link](https://semmle.com/careers) | ? |
+| SimSpace | Some | Full remote US | [Link](https://www.simspace.com/careers) | ? |
```
Comment (pbarry25) [inline README.md:147]: Example listing: https://www.simspace.com/careers/application?gh_jid=4370320006

File: README.md (hunk 2 above)
Comment (edoardottt) [reply README.md:34]: I can't spot any sign of remote working ..

File: README.md (hunk 3 above)
Comment (edoardottt) [reply README.md:96]: okay, thanks

File: README.md (hunk 4 above)
Comment (edoardottt) [reply README.md:147]: okay, thanks

**Review by edoardottt — COMMENTED** [review]
Are you sure about Bitdefender ?
There other 2 are okay

**Review by edoardottt — APPROVED** [review]
lgtm

**Comment by edoardottt** [comment]
thanks @pbarry25 !

**Pull request merged**